object-manager.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+define('MAUTIC_ROOT_DIR', __DIR__);
+
+// Fix for hosts that do not have date.timezone set, it will be reset based on users settings
+date_default_timezone_set('UTC');
+
+require_once 'autoload.php';
+
+use Mautic\CoreBundle\ErrorHandler\ErrorHandler;
+
+ErrorHandler::register('prod');
+
+$kernel   = new AppKernel('prod', false);
+$kernel->boot();
+
+return $kernel->getContainer()->get('doctrine')->getManager();
```

Comment:
This is a publicly accessible PHP endpoint. What it's for? Can we remove it from the production package?

Reply:
resolved in d1549732669b02a2a9729b1cbdf137623cd41a1b